src/Bicycle.java

```diff
@@ -1,30 +1,14 @@
-// Класс для велосипедов
-public class Bicycle implements Vehicle {
-    private String modelName;
-    private int wheelsCount;
-
-    @Override
-    public String getModelName() {
-        return modelName;
-    }
-
-    @Override
-    public void setModelName(String modelName) {
-        this.modelName = modelName;
-    }
-
-    @Override
-    public int getWheelsCount() {
-        return wheelsCount;
-    }
-
-    @Override
-    public void setWheelsCount(int wheelsCount) {
-        this.wheelsCount = wheelsCount;
+public class Bicycle extends Transport implements Vehicle {
+    public Bicycle(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
     }
 
     @Override
-    public void updateTyre() {
-        System.out.println("Меняем покрышку");
+    public void service() {
+        System.out.println("Обслуживаем велосипед " + getModelName());
+        int wheelsCount = getWheelsCount();
+        for (int i = 0; i < wheelsCount; i++) {
+            System.out.println("Меняем покрышку " + (i+1));
```

Comment:
```suggestion
            System.out.println("Меняем покрышку велосипеда " + (i+1));
```
чтобы подчеркнуть, что это не так, как для машины или грузовика, даже если код один и тот же